pkg/handlers/webhookreceiver.go

```diff
@@ -201,7 +202,9 @@ func (h *WebhookReceiverHandler) processAlert(alert template.Alert, mnl *oav1alp
 	if firing {
 		metrics.CountServiceLogSent(notification.Name, "firing")
 	} else {
-		metrics.CountServiceLogSent(notification.Name, "resolved")
+		if len(notification.ResolvedDesc) > 0 {
```

Comment:
Minor thing - as noted in the OAO PR, there's several occasions here of repeated logic about what constitutes when a resolved SL should be sent, so maybe it could be better captured as a function that's part of the ManagedNotification API which OA can leverage. That way if that logic should ever change (extremely unlikely in this case, but who knows), it need only be changed in one place.